apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/DatasetsResourceTest.java

```diff
@@ -3572,10 +3572,15 @@ Stream<Arguments> find__whenFilteringBySupportedFields__thenReturnMatchingRows()
                             FieldType.DICTIONARY, Operator.EQUAL, "sql_cost", "10")),
                     arguments(new ExperimentsComparisonFilter("meta_field",
                             FieldType.DICTIONARY, Operator.CONTAINS, "version[*]", "10")),
+                    arguments(new ExperimentsComparisonFilter("meta_field",
+                            FieldType.DICTIONARY, Operator.EQUAL, "version[*]", "[\"10\", \"11\", \"12\"]")),
                     arguments(new ExperimentsComparisonFilter("releases",
                             FieldType.DICTIONARY, Operator.CONTAINS, "$[1].version", "1.1")),
                     arguments(new ExperimentsComparisonFilter("meta_field",
-                            FieldType.DICTIONARY, Operator.CONTAINS, ".version[1]", "11")));
+                            FieldType.DICTIONARY, Operator.CONTAINS, ".version[1]", "11")),
+                    arguments(new ExperimentsComparisonFilter("releases",
+                            FieldType.DICTIONARY, Operator.CONTAINS, "$[*].fixes[1]", "11"))
```

Comment:
This test is wrong. The argument should be `$[*].fixes`or `$releases.fixes`. Otherwise, you're testing contains on a complex object (JSON array) but on the scalar value itself (11).